### DISCUSSION TITLE
applying freesurfer template-based registration to .arg sulcal file

### DISCUSSION THREAD

**Discussion by HdeV999** [brainvisa/brainvisa.github.io#179] [Q&A]
Dear BrainVISA team,
In order to assess the quality of a registration operated with freesurfer, my team and I are trying to visualize manually labelled sulci in the template-space. We have processed a cohort 1) with BrainVISA, and done manual labelling of a set of sulci, and 2) with FreeSurfer, and operated a registration to a given template (LH sym). We would like to visualize the .arg according to the registration to this specific template. Is there a way of doing that?
Thanks for any help on the matter!

**Comment by denisri** [comment]
Hi,
What is the output of the Freesurfer registration information ? Is is a volume-based registration, or a surface registration ?
If it's a volume-based registration, is it an affine one ? If yes we have to get and convert the transformation to Anatomist world.
Can you describe a bit all this please ?
Denis

**Reply by HdeV999** [reply]
It's a surface-based registration, using surfreg. It creates surf/?h.sphere.reg as an output.

**Comment by denisri** [comment]
Well, the problem is that this kind of surface matching allows to resample or deform a mesh, but does not bring information to transform data outside of the surface.
The only thing we can do is using a 3D volume transformation.

**Reply by HdeV999** [reply]
Ok, so if I understand correctly, the only way we can have relevant information to register the .arg file to the template in the same way as freesurfer is to use a volume-based registration in freesurfer instead of a surface-based one?

**Comment by denisri** [comment]
I'm not totally sure to understand what you want to do, but yes, the .arg contains data which come from the native volume and are not restricted to the cortical surface. So we need a transformation which is able to transform any point in space.
Freesurfer has an affine registration to the MNI space. Would this be enough ?

**Reply by HdeV999** [reply]
Makes sense, thanks. Not sure if this would allow us to answer our current question, we will need to give it a thought. Thanks!
